lib/katello/tasks/reimport.rake

```diff
@@ -32,7 +32,7 @@ namespace :katello do
       ack_key.import_pools
     end
 
```

Comment:
It would also be worth removing the `Dir.glob(Katello::Engine.root.to_s + '/app/models/katello/*.rb').each { |file| require file }` from this file. It errors out as well!

Reply:
Removed.